core/src/main/java/io/questdb/network/IODispatcherOsx.java

```diff
@@ -322,12 +325,20 @@ private boolean processRegistrations(long timestamp) {
                 keventWriter.readFD(suspendEvent.getFd(), eventId);
             }
 
-            if (operation == IOOperation.READ || context.getSocket().wantsTlsRead()) {
```

Comment:
nit: the `if` blocks order change here is irrelevant and not required for the fix.

Reply:
I am not sure I am following. can you elaborate please?

Reply:
Sorry for the confusion, I misread the changes in `processRegistrations`. Both changes are relevant.